site/config.toml

```diff
@@ -43,6 +43,12 @@ style = "emacs"
   Mastodon      = "mastodon.social/@fsfe"
   Diaspora      = ""
 
+DefaultContentLanguage = "en"
+defaultContentLanguageInSubdir = true
+[languages]
+[languages.en]
+  weight = 1
+
```

Comment:
I had to add some exception for HTML code in the config, seemingly caused by the 0.60 release of Hugo:

```
# enable raw HTML in MD files
[markup.goldmark.renderer]
unsafe = true
```

Reply:
Ok, looks fine.  I'm not familiar with those config options.